src/settings.rs

```diff
@@ -57,6 +57,18 @@ Settings which only apply to certain [`Planner`](crate::planner::Planner)s shoul
 #[derive(Debug, serde::Deserialize, serde::Serialize, Clone)]
 #[cfg_attr(feature = "cli", derive(clap::Parser))]
 pub struct CommonSettings {
+    /// Attribute the diagnostics of this run to a specific usage category
```

Comment:
```suggestion
    /// Relate the install diagnostic to a specific value
```

I dunno maybe like... this? Most of the existing copy describes 'an install' over 'a run'...

I am not really attached to this one.

Reply:
Yeah, it's a bit mushy any way you put it, haha. Done!